modules/reference/pages/cluster-properties.adoc

```diff
@@ -240,6 +264,18 @@ Enable automatic partition rebalancing when new nodes are added.
 
 ---
 
+=== enable_cluster_metadata_upload_loop
+
+Enables cluster metadata uploads. Required for xref:manage:tiered-storage.adoc#whole_cluster_restore_for_disaster_recovery[whole cluster restore].
```

Comment:
This link takes me to the Tiered Storage topic, not Whole Cluster Restore for Disaster Recovery.

Reply:
Fixed ✅ thanks